.github/workflows/ci.yml

```
@@ -267,6 +260,39 @@ jobs:
         cargo --version
         echo ""
 
+    - name: Install release version of PostgreSQL
+      if: matrix.version != 'pg16'
+      run: |
+        echo "----- Set up PostgreSQL Apt repository -----"
+        sudo apt-get install -y wget gnupg
+        sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+        wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+        sudo apt-get update -y -qq --fix-missing
+        echo ""
+
+        sudo apt-get install -y \
+          postgresql-$PG_VER \
+          postgresql-server-dev-$PG_VER
```

Makes this script pass shellcheck for `#!/bin/sh`:
```suggestion
          postgresql-"$PG_VER" \
          postgresql-server-dev-"$PG_VER"
```